examples/web_client/index.js

```diff
@@ -121,6 +122,26 @@ async function main () {
       }
     }, false)
 
+    document.addEventListener('mousedown', (e) => {
+      const BlockDistance = (bot.gamemode == 1) ? 8 : 5
+      const ButtonBlock = bot.blockAtCursor(BlockDistance)
+      if (!ButtonBlock) return
+      if (e.button === 0) {
+        if (bot.canDigBlock(ButtonBlock)) {
+          bot.dig(ButtonBlock)
```

Comment:
what happens when the user moves the cursor away from the block?

Reply:
I have no idea